docs/_book.yaml

```diff
@@ -27,92 +27,112 @@ upper_tabs:
   lower_tabs:
     # Subsite tabs
     other:
-    - name: "Tutorials"
+    - name: "Guides"
```

Comment:
I support moving the guide up, but think it should be "Guide" (singular) to keep it consistent with the other projects ("Guide & Tutorials"). There is one Cirq developer guide (that contains multiple sections and pages)

Reply:
Sounds good!